asdf/tags/core/external_reference.py

```diff
@@ -0,0 +1,64 @@
+from ...asdftypes import AsdfType
+
+
+class ExternalArrayReference(AsdfType):
```

Comment:
It's a mouthful, but just wondering whether this should be `ExternalArrayReferenceType` for consistency.

Reply:
Actually maybe not since this is both the tag type and the type itself, isn't it?